src/components/Property/Types/VectorProperty/ViewOptions/ColorVector.tsx

```diff
@@ -0,0 +1,38 @@
+import { Flex, RGBA } from '@mantine/core';
+
+import { ColorPicker } from '@/components/ColorPicker/ColorPicker';
+import { VectorPropertyProps } from '@/components/Property/Types/VectorProperty/VectorProperty';
+
+import { ValueList } from './DefaultValueList';
+
+export function ColorVector(props: VectorPropertyProps) {
+  const { disabled, setPropertyValue, value } = props;
```

Comment:
```suggestion
export function ColorVector({ disabled, setPropertyValue, value }: VectorPropertyProps) {

```

Reply:
Here I would prefer to keep the props object, to make it easy to spread to the value list view

The props handling for the vector types properties might need some discussion, in general

Reply:
@engbergandreas also commented on this. Let's discuss it :)